package.json

```diff
@@ -183,7 +183,7 @@
     "react-responsive-decorator": "^0.0.1",
     "react-router": "^4.2.0",
     "react-sizeme": "^2.4.4",
-    "react-slick": "^0.23.1",
+    "react-slick": "^0.23.2",
```

Comment:
React-slick is pretty fragile -- can you be sure to check the new artist page and Recently Viewed slider in storybooks before merging?

Reply:
Yep already checked out all the existing slideshows, looking good.